### DISCUSSION TITLE
Join Character not working on first text

### DISCUSSION THREAD

**Discussion by adelinemdc** [dialogic-godot/dialogic#796] [Q&A] [answered]
Hello! I'm having an issue where the characters portraits aren't showing up until the second time they speak. I am using the join event, and they do join the second time they speak, just not the first.

![dialogic_screen](https://user-images.githubusercontent.com/100298848/155358631-8c52335a-62d7-45b7-bd69-80f7cf2ba832.png)

This is the code I'm using. I have a separate unpausing function for interactions that I'm using here, though when I use the standard dialogic code the portraits do the same thing:

> `extends Area2D
> 
> signal interactFinished
> 
> func interact():
> 	var dialog = Dialogic.start('hemoDialog_Alder')
> 	dialog.pause_mode = Node.PAUSE_MODE_PROCESS
> 	dialog.connect('timeline_end', self, 'unpause')
> 	add_child(dialog)
> 
> func unpause(timeline_name):
> 	emit_signal("interactFinished")
> `

**Comment by coppolaemilio** [comment, marked as answer]
What version of Dialogic are you using? The 1.4 beta? or a previous version

**Reply by adelinemdc** [reply]
I'm using 1.4

**Reply by Jowan-Spooner** [reply]
when did you download? This doesn't look like it's the beta. 1.4 is still WIP, so it changes regularly. Try downloading the beta or even the main branch again.

**Reply by adelinemdc** [reply]
I downloaded it the beginning of this month. I'll try downloading again, thank you!

**Reply by adelinemdc** [reply]
- Just to add on to this, I checked the changelog and it says it's the 1.4 wip

**Reply by coppolaemilio** [reply]
Yes, try to download the latest one and see if it works: https://github.com/coppolaemilio/dialogic/archive/refs/heads/main.zip

**Reply by adelinemdc** [reply]
That did indeed fix the problem! Thank you so much!

**Reply by sshearer** [reply]
I had/have the same issue in v1.3, with no custom coding.  I just added an empty dialog.
